client/src/main/java/gui/building/FleetDropController.java

```diff
@@ -150,12 +165,16 @@ private void setupShipBoardUpdater() {
 
   private final EventHandler<MouseEvent> connectWhenClicked =
       event -> {
-        connectButton.setVisible(false);
-        setupClient();
-        Player player = new Player(fleet , userName.getText());
-        FleetSender fleetSender = new FleetSender(getClient(), player);
-        fleetSender.sendFleetToServer();
-        nextButton.fireEvent(new ConnectEvent());
+        if (!fleet.isFleetEmpty()) {
+          setupClient();
+          Player player = new Player(fleet, userName.getText());
+          FleetSender fleetSender = new FleetSender(getClient(), player);
+          fleetSender.sendFleetToServer();
+          nextButton.fireEvent(new ConnectEvent());
+        } else {
+          info.setText("You can't play with empty fleet");
```

Comment:
Hardcoded value. That will be a problem in language choice system.

Reply:
done